### DISCUSSION TITLE
프로젝트 #2: KAITA 멤버 디렉토리 서비스

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#21] [3. 프로젝트 아이디어]
설명: 멤버분들이 해외에 계신 분들도 있고 다양한 지역에 있는 분들이 많은데 그 분들이 좀 더 정리 되어 있는 웹사이트가 있으면 도움이 될것 같다. 

테크 스택: 웹 기반 (HTML, CSS, Javascript, etc), 모바일 (안드로이드, iOS)

**Comment by klee5760** [comment]
PC-모바일 배우기 위한 콤비네이션 프로젝터 좋은 생각입니다. 앞으로 이런 프로젝터가 많이 필요합니다.
